gdc_rnaseq_tools/augment_star_counts.py

```diff
@@ -0,0 +1,341 @@
+import logging
+from argparse import Namespace
+from enum import Enum
+from typing import List, Optional, Text, Union
+
+import numpy as np
+import pandas as pd
+
+from gdc_rnaseq_tools.utils import DataError, DataFormatError, get_logger
+from tests import FakeArgs
+
+
+class ColumnNames(Enum):
+    @classmethod
+    def cols(cls) -> List[Text]:
+        '''
+        Returns an ordered list of Enum members. Designed to return the
+        columns in a specific order for purposes of reading or printing.
+
+        Args:
+            None
+        
+        Returns:
+            list of column names as strings
+        '''
+        return [member.value for member in cls]
+
+
+class CountsColumns(ColumnNames):
+    GENE_ID = 'gene_id'
+    UNSTRANDED = 'unstranded'
+    STRANDED_FIRST = 'stranded_first'
+    STRANDED_SECOND = 'stranded_second'
+
+
+class GeneInfoColumns(ColumnNames):
```

Comment:
These look better.

Something to think about: if you weren't using pandas for the file reading, these Enums would work better as typed NamedTuples

```python
from typing import NamedTuple

class GeneInfoNT(NamedTuple):
    gene_id: str
    total_exon_length: int
    gene_name: str
    gene_type: str
    chromosome: str
```

With a NamedTuple you'd be able to save each record you read into this data type and you'll still be able to get the list of columns via `GeneInfoNT._fields`